src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -154,14 +162,29 @@ def configure_well_initial_conditions(self, alfacase: CaseDescription) -> None:
             ),
         )
 
-    # TODO PWPA-1996: review the configurations done here.
     def configure_physics(self, alfacase: CaseDescription) -> None:
```

Comment:
At a glance, it seems we don't need to overwrite this method.
```
    def configure_physics(self, alfacase: CaseDescription) -> None:
        """Configure the description for the physics data."""
        alfacase.physics = PhysicsDescription(
            hydrodynamic_model= HydrodynamicModelType.FourFields if self.has_water() else HydrodynamicModelType.ThreeLayersGasOilWater,
            energy_model=EnergyModel.GlobalModel,
			simulation_regime=SimulationRegimeType.Transient if self.has_gas_lift() else SimulationRegimeType.SteadyState,
			initial_condition_strategy=InitialConditionStrategyType.Constant,
        )
```

But only `has_water` and `has_gas_lift`, thus the code will be much more simple.

Reply:
One reason I splited these two methods is because I don't have the `has_gas_lift` in the base class cause the data to build it is just available for the production operation file and I it will be weird in the base once I think it only makes sense for the `ProductionOperationBuilder`. However, I think I can simplify and remove the method from the `InjectionOperationBuilder` if I set it in the base like

```python
    def configure_physics(self, alfacase: CaseDescription) -> None:
        """Configure the description for the physics data."""
        alfacase.physics = PhysicsDescription(
            hydrodynamic_model=HydrodynamicModelType.ThreeLayersGasOilWater,
            energy_model=EnergyModel.GlobalModel,
            # moved parameters from InjectionOperationBuilder
            simulation_regime=SimulationRegimeType.SteadyState,
            initial_condition_strategy=InitialConditionStrategyType.Constant,
        )
```
It feels a little bit simpler to me, What do you think?

Reply:
Done.

Reply:
It is not weird `BaseOperationBuilder` has a `has_gas_lift` returning `False` and it be overwritten in `ProductionOperationBuilder` with the correct checks. The same for the `has_water`.
IMHO the code could be more clear and straight as I suggested. But it is up to you to decide it.

Reply:
ok.